### ISSUE TITLE
[USER STORY] Read list of all outstanding/unpaid invoices

### ISSUE THREAD

**Issue by data-david** [Informasjonsforvaltning/behov#127]
As: a credit agency/bank application (PoC)
I wish to: read list of all outstanding/unpaid invoices
So that: I can evaluate amount of unpaid invoices and give credit accordingly

Related to epic: #4 

Link to design: 

Link to solution architecture:

**Comment by ekoskent** [comment]
I think we should revise the data need with a bank as their needs for credit risk analytics might be broader.

**Comment by data-david** [comment]
Agreed, but let's then make another user story. As an MVP with at least this scope can be useful for the development.

**Comment by ekoskent** [comment]
This API does not to be realized as such. The Informasjonsforvaltning/behov#269 "get transaction" API will work here too. To be estimated in the later times together with the possible implementation of the "search API".
